src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,86 @@
+package namor.template.calculator;
+
+import java.util.LinkedList;
+
+public class ClassicCalculator {
+
+    private LinkedList<String> operands;
+
+    public ClassicCalculator() {
+        this.operands = new LinkedList<>();
+    }
+
+    public double calculate(String arithmeticExpression) {
+
+        for (String operationElement : arithmeticExpression.split(" ")) {
+            operands.addLast(operationElement);
+        }
+
```

Comment:
I want you to make the unmodifiable list. So just make `List<String> someName = Collections.unmodifiableList(operands)` and proceed to work with your immutable list. It will force you to do few good changes in the code